fancy.MD

```diff
@@ -0,0 +1,7 @@
+### Super fancy
+
+           mmm          mmm          mmm          mmm
+      mmm  | |     mmm  | |     mmm  | |     mmm  | |
+      | |  | |     | |  | |     | |  | |     | |  | |
+      | |  \) \    | |  \) \    | |  \) \    | |  \) \
+      \) \  \__)   \) \  \__)   \) \  \__)   \) \  \__)
```

Comment:
@idt911-jmiller  Do you mind if I make them more fancy in some commit down the road?